Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1

```diff
@@ -153,17 +153,19 @@ function Run-Command
 	{
         if( $psversiontable.PSVersion.Major -le 4)
         {
-           cmd.exe /c "`"$command`""
+           cmd.exe /c "`"$command`"" 2>&1
         }
         else
         {
-           cmd.exe /c "$command"
+           cmd.exe /c "$command" 2>&1
         }
 
     }
 	catch [System.Exception]
     {
-        throw $_.Exception.Message    
+        $exception = $_.Exception
+        Write-Verbose "Error occured is $($exception.Message)"
```

Comment:
Just for my understanding we are trying to log exception message here? why can't we directly log $_.Exception.Message